afanasy/src/watch/itemjobtask.h

```diff
@@ -15,7 +15,11 @@ class ItemJobTask : public Item
 {
 	Q_OBJECT
 public:
+	/// main ctor, used when adding a task to a ListTask
 	ItemJobTask( ListTasks * i_list, const ItemJobBlock * i_block, int i_numtask, const af::BlockData * i_bdata);
+	/// ctor used to generate context menu, not to add to a list.
+	/// It is just holding information about job/block/task id.
+	ItemJobTask( int i_job_id, int i_block_num, int i_task_num, const QString &itemname, QWidget *parent);
```

Comment:
This I'm not really proud of. It makes the code use `ItemJobTask` for two different purposes a) rendering in lists and b) just generating ctx menu. The problem is that in the second case we have only partial information about the Task.